notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -816,6 +816,49 @@ coping with the death of a colleague [The command crew mourned the death of Data
 space dock [The Enterprise was shown being repaired in a giant space dock in orbit around the earth at the conclusion of the film.]
 
 
+movie: Godzilla Against Mechagodzilla (2002)
+============================================
+
+:: Title
+Godzilla Against Mechagodzilla
+
+:: Description
+Godzilla Against Mechagodzilla (ゴジラ×メカゴジラ, Gojira tai Mekagojira,
+released in Japan as Godzilla vs. Mechagodzilla) is a 2002 Japanese kaiju
+film directed by Masaaki Tezuka, written by Wataru Mimura, and produced by
+Shogo Tomiyama. It is the 27th film in the Godzilla franchise and the fourth
+film in the franchise's Millennium period.
+
+:: Date
+2002-12-14
+
+:: References
+https://en.wikipedia.org/wiki/Godzilla_Against_Mechagodzilla
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+giant monster [Godzilla was terrorizing the people of Japan.]
+human vs. monster [The giant bio-mechanical robot Mechagodzilla was used by the Japan Self-Defense Forces to defend the Japan from a rampaging Godzilla.]
+father and daughter [Central to the story was the relationship between Dr. Yuhara and his young daughter Sara who was struggling to get over the recent passing of her mother.]
+speculative weapon [A team of Japanese scientists led by Dr. Yuhara created the giant bio-mechanical robot Mechagodzilla as a "anti-Godzilla super weapon". The question of whether Mechagodzilla might also be used as a weapon of war was also discussed.]
```

Comment:
Hmm, this opens a can of worms. I don't think we categorise similar things as weapons. If we start, are tanks and battleships and space fighters (x-wings) also weapons then? 

Sure the may have used the word "weapon" in the story. That is neither here nor there because that word is frequently used as a figure of speech. If I say that he wielded his pen as a weapon, should we theme his pen as a weapon???

Reply:
I had similar reservations. I took it out and pushed changes.